### DISCUSSION TITLE
Engrave in headless mode

### DISCUSSION THREAD

**Discussion by DomonkosSuranyi** [arkypita/LaserGRBL#1936] [Q&A]
I want to start the engraving from a script.
I mean that I have 3 SVG files (line drawing) and a well tested configuration for each one.

Example dream scenario:
I have an easy-to-use fab UI, where I can click on one of the three pictures, and the engraver get to work.

Does LaserGRBL have a headless mode or provides an API to make a similar scenario possible?

**Comment by DomonkosSuranyi** [comment]
I guess, I got it.
I don't need LaserGRBL at all for that job, because if I can generate the gcode from the svg files, I can send this gcode directly to the engraver

**Reply by DomonkosSuranyi** [reply]
But I'd be happy if someone would confim that

**Reply by Tribo-2dot0** [reply]
Podes usar LaserGrbl para gerar o gcode e depois salva-lo.
